test/functional/tests/fault_injection/test_standby.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright(c) 2019-2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import cli, casadm
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_utils.size import Size, Unit
+from api.cas.cache_config import CacheLineSize, CacheMode, CacheStatus
+from test_tools.dd import Dd
+from api.cas.cli import standby_activate_cmd
+from api.cas.cli_messages import check_stderr_msg, activate_with_different_cache_id
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_activate_neg_cache_id():
+    """
+    title: Blocking cache with mismatching cache ID activation
+    description: |
+      Try restoring cache operations from a replicated cache that was initialized
+      with different cache ID than the original cache
+    pass_criteria:
+      - The activation is cancelled
+      - The cache remains in Standby detached state after an unsuccessful activation
+      - The cache exported object is present after an unsuccessful activation
+      - A proper error message is displayed
+    """
+    with TestRun.step("Prepare two partitions of the same size"):
+        disk = TestRun.disks["cache_1"]
+        disk.create_partitions([Size(200, Unit.MebiByte), Size(200, Unit.MebiByte)])
+        active_dev = disk.partitions[0]
+        passive_dev = disk.partitions[1]
```

Comment:
I'm not sure if passive is best description. I wonder if we should use active and standby names in all tests.
What do you think?